mm2src/coins/solana/solana_tests.rs

```diff
@@ -124,6 +125,84 @@ mod tests {
         assert_eq!(res.is_valid, false);
     }
 
+    #[tokio::test]
+    #[cfg(not(target_arch = "wasm32"))]
+    async fn solana_transaction_simulations() {
+        let passphrase = "federal stay trigger hour exist success game vapor become comfort action phone bright ill target wild nasty crumble dune close rare fabric hen iron".to_string();
+        let (_, sol_coin) = solana_coin_for_test(passphrase.clone(), SolanaNet::Devnet);
+        let valid_tx_details = sol_coin
+            .withdraw(WithdrawRequest {
+                coin: "SOL".to_string(),
+                from: None,
+                to: sol_coin.my_address.clone(),
+                amount: BigDecimal::from(0.0001),
+                max: false,
+                fee: None,
+            })
+            .compat()
+            .await
+            .unwrap();
+        assert_eq!(valid_tx_details.spent_by_me, BigDecimal::from_str("0.000105").unwrap());
```

Comment:
Is the fee `0.000005` always? How about using mockable functions to be sure?

Reply:
> Is the fee `0.000005` always? How about using mockable functions to be sure?

The fees is always equal to `coin.estimate_withdraw_fees()` I can use that instead of the mock.